package.json

```diff
@@ -9,8 +9,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "ethereum-multicall": "^2.9.0",
+    "@notional-finance/sdk-v2": "^0.4.15",
```

Comment:
notifications need to be implemented using the available packages within the project.
either web3, ethereum-multicall or node-fetch